### PR TITLE
For v3.0 release

### PR DESCRIPTION
**Pull request by bmeyers**
Trying to help out, if possible...

### CODE REVIEW

**Comment by bmeyers** [comment]
@mikofski Just realized I got the values wrong for the quickstart. I'll rerun those.

**Comment by bmeyers** [comment]
@mikofski read for review

File: pvmismatch/pvmismatch_lib/pvmodule.py [hunk 1]
```diff
@@ -289,6 +289,76 @@ def setSuns(self, Ee, cells=None):
...
+                        self.pvcells[cell_id] = old_pvcells[pvcell]
+            elif np.size(Tc) == Ncells:
+                # Find unique irradiance values
+                # TODO possible "cleaner" alternative by grouping cells into tuples that match the set temp
+                # E.g: pvsys.setTemps({X: {Y: {'Tc': (280, 290), 'cells': [(2, 3), 17]}}})
+                cells = np.array(cells)
+                Tc = np.array(Tc)
+                unique_tc = np.unique(Tc)
+                for a_Tc in unique_tc:
+                    cells_subset = cells[np.where(Tc == a_Tc)]
```
Comment (mikofski) [inline pvmismatch/pvmismatch_lib/pvmodule.py]: just use logical indexing

    cells_subset = cells[Tc == a_Tc]

**Comment by bmeyers** [comment]
Thanks Mark!

**Pull request merged**